src/main/scala/com/codacy/duplication/pmd/Cpd.scala

```diff
@@ -98,31 +109,38 @@ object Cpd extends DuplicationTool {
           ScalaTokenizer,
           cpdScala.getExtensions.asScala: _*) {}
         Some(cpdConfiguration(scalaLanguage, 50, options))
-      case Languages.CSharp => Some(cpdConfiguration(new CsLanguage, 50, options))
-      case _                => None
+      case _ => None
     }
   }
 
-  private def cpdConfiguration(cpdLanguage: CPDLanguage,
-                               defaultMinToken: Int,
-                               options: Map[DuplicationConfiguration.Key, DuplicationConfiguration.Value]) = {
+  // TODO: Move to codacy-plugins-api
+  implicit def boolean(value: JsValue): Option[Boolean] =
+    Option(value: JsValue).collect { case JsBoolean(bool) => bool }
 
-    val ignoreAnnotations = true
-    val skipLexicalErrors = true
+  implicit def int(value: JsValue): Option[Int] =
+    Option(value: JsValue).collect { case JsNumber(bigDecimal) => bigDecimal.toInt }
 
-    val minTokenMatch: Int = options.get(DuplicationConfiguration.Key("minTokenMatch")).fold(defaultMinToken) {
-      value: DuplicationConfiguration.Value =>
-        Option(value: JsValue).collect {
-          case JsNumber(number) => number.toInt
-          case _                => defaultMinToken
-        }.getOrElse(defaultMinToken)
+  implicit class DuplicationConfigurationExtended(
```

Comment:
Is this implicit that important when we only use this method in one place? 
Why not have just a simple local method?
Either way I would prefer for it to not be here in the middle of the class and maybe be in it's own file or something similar.